### DISCUSSION TITLE
Bottoni Ikea

### DISCUSSION THREAD

**Discussion by zolabus** [indomus/forum#1669] [HUB: Home Assistant]
Ciao a tutti ho acquistato due bottoni ikea un on/off e un bottone "singolo", ho un hub sonoff zigbee.
Pur riuscendo (con molta fatica) a far riconoscere i pulsanti, non riesco di fatto a farli funzionare nè con i blueprint ne con le automazioni in quanto pur funzionando le prime volte poi i pulsanti sembrano non essere più "vivi" e smettono di rispondere ai comandi...
Qualche spunto? meglio acquistare altri pulsanti?
Grazie

**Comment by Scialla** [comment]
io li uso con una chiavetta e funzionano benissimo

**Reply by zolabus** [reply]
che chiavetta? con blueprint oo automaizoni?
Ho letto che hanno problemi con le batterie che le fanno esaurire presto

**Reply by Scialla** [reply]
io uso una chiavetta con chip 2538; blueprint o automazioni cambia ben poco. Posso dirti che io li uso con zigbee2mqtt, tu non hai specificato se usi z2m o zha.
Io ho due telecomandi a 4 pulsanti, uno a due pulsanti, due a pulsante singolo, anche un sensore presenza. Tutti funzionano correttamente e le batterie sono su da mesi, alcuni da anni..
Hai router per casa? non è che perdono il segnale?
Ti consiglio di mettere a debug per vedere se arriva qualcosa quando premi i pulsanti, magari fai prove vicino al coordinator

**Reply by zolabus** [reply]
uso zha coem faccio ad usare zigbee2mqtt?
comuque se clicco sul bottone negli eventi zha non vedo niente...vedo solo quando resetto tutto e reimporto i bottoni..allora funzionano per un po' poi smette

**Reply by snakuzzo** [reply]
per quanto riguarda zigbee2mqtt -> ti basta cercare, anche sul nostro sito, e trovi tutte le info.
Quanto alla richiesta iniziale...non so se ho capito bene quello che ti serve, ma una cosa è certa...
non puoi dare i comandi (es. pressione pulsante) da HA. I device a batteria vanno in deep sleep (così come hai detto anche tu).
Su HA devi fare il giro inverso, ovvero a fronte della pressiona fisica del tasto, intercettare l'evento e fare altre cose.

**Comment by zolabus** [comment]
si in pratica mi serve che il tasto spenga una luce da fuori casa visto che quando apro la porta la sera si accende la luce (anche quando esco però)...
mi dai un aiuto?
Coem detto sembra che non arrivi nessun evento zha...

**Reply by snakuzzo** [reply]
Se premi il tasto fisico e non arriva niente su HA/ZHA...non è che si possa fare tanto.
